manage_user_page.php

```diff
@@ -409,7 +416,8 @@
 				</td>
 				<td><?php echo string_display_line( $v_realname ) ?></td>
 				<td><?php
-					if( $t_unique_emails && !user_is_email_unique( $v_email, $v_id ) ) {
+					# Display warning icon if emails should be unique and a duplicate exists
+					if( array_key_exists( $v_email, $t_duplicate_emails ) ) {
```

Comment:
Will not work with email addresses with uppercase characters in it,  as user_get_duplicate_emails() delivers lowercase email addresses.

Reply:
Good catch, thanks ! Will fix